src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxProcessingDaoJdbc.java

```diff
@@ -166,13 +174,24 @@ public void doTaxProcessing(String taxType, int reportYear, java.sql.Date startD
         });
     }
 
-    private String getForeignDraftAndWireTransferSelectSql(int paramSize, TransactionDetailSummary summary) {
-        return TaxSqlUtils.getQueryWithoutColumnPrefixes(SqlText.SELECT, summary.dvRow.dvDocumentNumber,
-                SqlText.FROM, summary.dvRow.tables.get(TaxSqlUtils.getTableIndexForField(summary.dvRow.dvDocumentNumber)),
-                SqlText.WHERE,
-                        TaxSqlUtils.getInListCriteria(summary.dvRow.dvDocumentNumber, paramSize, true, false),
-                SqlText.AND,
-                        summary.dvRow.documentDisbVchrPaymentMethodCode, SqlText.IN, "('F','W')");
+    private String getForeignDraftAndWireTransferSelectSql(int paramSize, TransactionDetailSummary summary, String docType) {
+    	 if(DisbursementVoucherConstants.DOCUMENT_TYPE_CODE.equalsIgnoreCase(docType)){
+    		return TaxSqlUtils.getQueryWithoutColumnPrefixes(SqlText.SELECT, summary.dvRow.dvDocumentNumber,
+                    SqlText.FROM, summary.dvRow.tables.get(TaxSqlUtils.getTableIndexForField(summary.dvRow.dvDocumentNumber)),
+                    SqlText.WHERE,
+                            TaxSqlUtils.getInListCriteria(summary.dvRow.dvDocumentNumber, paramSize, true, false),
+                    SqlText.AND,
+                            summary.dvRow.documentDisbVchrPaymentMethodCode, SqlText.IN, "('F','W')");
+    	}
+    	else if(PurapConstants.PurapDocTypeCodes.PAYMENT_REQUEST_DOCUMENT.equalsIgnoreCase(docType)){
+    		return TaxSqlUtils.getQueryWithoutColumnPrefixes(SqlText.SELECT, summary.prncRow.preqDocumentNumber,
+    				SqlText.FROM, summary.prncRow.tables.get(TaxSqlUtils.getTableIndexForField(summary.prncRow.preqDocumentNumber)),
+    				SqlText.WHERE,
+    				TaxSqlUtils.getInListCriteria(summary.prncRow.preqDocumentNumber, paramSize, true, false),
+    				SqlText.AND,
+    				summary.prncRow.paymentMethodCode, SqlText.IN, "('F','W')");
+    	}
+    	else return StringUtils.EMPTY;

```

Comment:
I would suggest adding curly braces to the return statement in the "else" block.